pkgs/applications/networking/irc/weechat/scripts/weechat-matrix/default.nix

```diff
@@ -22,22 +22,32 @@ let
     python-magic
   ]);
 
+  owner = "poljar";
+  repo = "weechat-matrix";
```

Comment:
There's very little to be gained from having these in variables, and it adds overhead when reading the code.

Reply:
changed!